src/network/clientpackethandler.cpp

```diff
@@ -515,7 +515,7 @@ void Client::handleCommand_Movement(NetworkPacket* pkt)
 	player->movement_gravity                = g * BS;
 }
 
-void Client::handleCommand_Fov(NetworkPacket* pkt)
+void Client::handleCommand_Fov(NetworkPacket *pkt)
```

Comment:
You could make the same change in the header file

Reply:
Done